src/test/java/com/artipie/npm/NpmITCase.java

```diff
@@ -89,7 +89,7 @@
     /**
      * Helper for url that may contains user credentials.
      */
-    private UrlCredsHelper urlraw;
+    private RepositoryUrl url;
```

Comment:
@genryxy please update comment for `url` fields in tests and we will be good to merge